teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, name = 'unknown')
+    super(age, name)
+    @specialization = specialization
+  end

```

Comment:
- [ ] Kindly add the `age` and `parent_permission` parameters in the `initialize` method.

- [ ] Kindly add the missing `parent_permission` to the `super` method arguments.